tests/wycheproof/test_rsa.py

```diff
@@ -138,25 +138,32 @@ def test_rsa_pkcs1v15_signature_generation(backend, wycheproof):
 )
 def test_rsa_pss_signature(backend, wycheproof):
     digest = _DIGESTS[wycheproof.testgroup["sha"]]
-    if backend._fips_enabled and isinstance(digest, hashes.SHA1):
-        pytest.skip("Invalid params for FIPS. SHA1 is disallowed")
-
-    key = wycheproof.cache_value_to_group(
-        "cached_key",
-        lambda: serialization.load_der_public_key(
-            binascii.unhexlify(wycheproof.testgroup["keyDer"]),
-        ),
-    )
-    assert isinstance(key, rsa.RSAPublicKey)
     mgf_digest = _DIGESTS[wycheproof.testgroup["mgfSha"]]
-
     if digest is None or mgf_digest is None:
         pytest.skip(
             "PSS with digest={} and MGF digest={} not supported".format(
                 wycheproof.testgroup["sha"],
                 wycheproof.testgroup["mgfSha"],
             )
         )
+    salt_length = int(wycheproof.testgroup["sLen"])
```

Comment:
is `int()` required here? if so, why isnt' it required below -- and can you reuse teh local variable down below.

Reply:
Fixed (it's not required so we don't need this var at all)